pkg/server/apis/v1alpha1/types.go

```diff
@@ -203,3 +203,45 @@ type TLSClientConfig struct {
 	// CAData takes precedence over CAFile
 	CAData []byte `json:"caData,omitempty" bson:"caData"`
 }
+
+// StatusStats represents statistics of project or workflow.
+type StatusStats struct {
```

Comment:
Stats not only include status, but also workflow count. What about `Stats`? We can add more infos into it later.